src/config.h.in

```diff
@@ -208,24 +208,11 @@ inline int Ctz(unsigned __int64 mask) {
 #endif
 }
 
-#if _M_ARM64
-//https://stackoverflow.com/a/70012905
-template <typename T>
-int BrianKernighanPopcount(T value) {
-  int count;
-  for(count = 0; value; count++)
-  {
-    value &= value - 1;
-  }
-  return count;
-}
-#endif
-
 inline int Popcount(unsigned long value) {
   #if _M_X64 || _M_IX86
   return __popcnt(value);
 #elif _M_ARM64
-  return BrianKernighanPopcount(value);
+  return _CountOneBits(value);
```

Comment:
Seems reasonable.   I wonder why MSVC has two names for the same thing depending on the architecture?

Reply:
I'm MSVC too. The `BrianKernighanPopcount` was added by me 1+ years ago. At that moment I just found `__popcnt` not exists on ARM64, so I just added the pure software version. But today I'm awared of the `_CountOneBits` by accident.